_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: Analyze Jaeger trace data 
+parent: Trace analytics
+nav_order: 55
+---
+
+# Analyze Jaeger trace data
+
+Introduced 2.5
+{: .label .label-purple }
+
+Trace analytics now supports Jaeger trace data. Jaeger can store its trace data in OpenSearch, but users that use OpenSearch as their backend for Jaeger data can now benefit from trace analytics capability.
+
+When you perform trace analytics, you can select from two data sources:
+
+- **Data Prepper** – Data ingested into OpenSearch through Data Prepper. This data source requires OpenTelemetry (OTEL) index type.
+- **Jaeger** – Trace data stored within OpenSearch as its backend. This data source requires the Jaeger index type.
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the trace analytics capability. When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the Dashboards.
+
+Jaeger trace data that you can analyze includes span data, service dependency data, and service and operation endpoint data. Jaeger span data analysis requires some configuration.
```

Comment:
What is service dependency data - not sure if we have that, maybe word in a different way?

Reply:
Maybe want to add here limitation that errors need to have this flag set: --es.tags-as-fields.all=true

See this comment: https://github.com/opensearch-project/dashboards-observability/issues/83#issuecomment-1370084286

Reply:
This is also available in the product, but maybe deserves a special call out that they need to have indices in the format of *jaeger-span-* and *jaeger-service*

Reply:
Could you also specify which file they set this flag in?

Reply:
The flag is set as a configuration for ingestion

Reply:
thanks. also added the procedure you shared with me.